packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html

```diff
@@ -0,0 +1,138 @@
+<oui-field
+    label="{{:: 'hosting_multisite_domain_configuration_home' | translate }}"
+    size="l"
+    error-messages="{
+        pattern: ('hosting_multisite_domain_configuration_path_help' | translate)
+    }"
+>
+    <div class="oui-input-group">
+        <span class="oui-button oui-button_secondary">./</span>
+        <input
+            type="text"
+            class="oui-input"
+            id="domainAttacheModeOvhFormSelectedPath"
+            name="selectedPath"
+            placeholder="{{:: 'hosting_multisite_domain_configuration_myfolder' | translate }}"
+            data-ng-model="$ctrl.path"
+            data-ng-pattern="$ctrl.VALID_PATH"
+        />
+    </div>
+</oui-field>
+<oui-field
+    ng-if="$ctrl.hosting.isCloudWeb"
+    label="{{:: 'hosting_multisite_domain_configuration_runtime' | translate }}"
+    data-size="l"
+>
+    <oui-select
+        data-items="$ctrl.runtimes"
+        data-model="$ctrl.runtime"
+        name="runtime"
+        match="name"
+    >
+        <span ng-bind="$item.name"></span>
```

Comment:
👀 On custom elements we tend to drop the `data` prefix but we not used to apply
this rule to the native HTMLElement. Is it intentional?

Reply:
Oopsie ! I think I mixed both rules, I've updated it